nautobot/extras/tests/test_forms.py

```diff
@@ -1095,29 +1091,30 @@ def test_create_webhooks_with_same_content_type_same_url_common_action(self):
         )
 
 
+@skip(reason="Skipping until we have items that need to be deprecated again.")
```

Comment:
Although this test is no longer relevant, I thought it would be nice to retain the framework and just skip it for now in case we want to reuse it in the future.